src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using GitHub.DistributedTask.Pipelines;
 using GitHub.DistributedTask.WebApi;
 using GitHub.Runner.Common.Util;
+using GitHub.Runner.Worker.Container;
```

Comment:
despite the diff, the actual change is pretty small. The diff looks bad due to the signature change of IActionCommandExtension.ProcessCommand()